scripts/zones/Garlaige_Citadel/Zone.lua

```diff
@@ -56,24 +59,26 @@ zoneObject.onConquestUpdate = function(zone, updatetype, influence, owner, ranki
 end
 
 zoneObject.onTriggerAreaEnter = function(player, triggerArea)
-    local triggerAreaID = triggerArea:GetTriggerAreaID()
-    local leverSet = math.floor(triggerAreaID / 9)                      -- The set of levers player is standing on (0, 1, 2)
-    local gateId   = ID.npc.BANISHING_GATE_OFFSET + (9 * leverSet) -- The ID of the related gate
-
-    -- Logic when standing on the lever.
-    GetNPCByID(ID.npc.BANISHING_GATE_OFFSET + triggerAreaID):setAnimation(xi.anim.OPEN_DOOR)
-
-    -- If all 4 levers of a set are down, open related gate for 30 seconds.
-    if
-        GetNPCByID(gateId):getAnimation() == xi.anim.CLOSE_DOOR and -- Avoid spamming.
-
-        GetNPCByID(gateId + 1):getAnimation() == xi.anim.OPEN_DOOR and
-        GetNPCByID(gateId + 2):getAnimation() == xi.anim.OPEN_DOOR and
-        GetNPCByID(gateId + 3):getAnimation() == xi.anim.OPEN_DOOR and
-        GetNPCByID(gateId + 4):getAnimation() == xi.anim.OPEN_DOOR
-    then
-        player:messageSpecial(ID.text.BANISHING_GATES + leverSet)
-        GetNPCByID(gateId):openDoor(30)
+    if triggerArea:GetTriggerAreaID() ~= 30 then
```

Comment:
this might be better to check if trigger area is <= 22. Rather than making 30 the edge case.

That way any new trigger areas won't have to be added to this if statement. You can also feel free to make the trigger area you're creating 23.